### ISSUE TITLE
Bugfix: Execute manual_control.py without display

### ISSUE THREAD

**Issue by Satadru-Banerjee** [carla-simulator/scenario_runner#880]
While working on setting up carla on an AWS instance with ubuntu, we faced an error that manual_control.py would not execute as it was missing a display(Working on CLI only). There were no official solutions available on the scenario_runner git yet. So this seems like a quick fix.

Soln: Added a dummy display so code can run without a display. Just add the following line in the game_loop(args) method

 os.environ["SDL_VIDEODRIVER"] = "dummy"

tldr: Manual control does not run on carla headless mode in CLI due to lack of display. Added code snippet to provide a dummy display for the script execution.

device: AWS EC2 g4dn.xlarge
OS: Ubuntu
RAM: 16GB
GPU: 15 GB Nvidia Tesla T4
